app/src/main/java/com/mho/training/data/translators/GenreDataTranslator.kt

```diff
@@ -0,0 +1,9 @@
+package com.mho.training.data.translators
+
+import com.example.android.domain.Genre
```

Comment:
You could use import alias instead of set a different name for each class
```suggestion
import com.example.android.domain.Genre as DomainGenre
import com.example.android.framework.data.remote.models.genre.Genre as ServerGenre
```

Reply:
I prefer don't use alias in this case because I use domain entities with any prefix or suffix, only for other layers